Contributions.md

```diff
@@ -0,0 +1,66 @@
+# Contributions
+
+This document list current and past contributions made to standards that has some relationship with the jQuery Foundation projects.
+
+# W3C
+
+## Pointer Events
+
+- [Recommendation 24 February 2015](https://www.w3.org/TR/pointerevents/)
+- [Editor's draft](https://w3c.github.io/pointerevents/)
+- participants: @scottgonzalez
+
+## Touch Events
+
+- [Recommendation 10 October 2013](https://www.w3.org/TR/touch-events/)
+- [Editor's draft](https://w3c.github.io/touch-events/)
+- participants: @scottgonzalez
+
+## Other contributions
+
+- [fixed the behavior](https://lists.w3.org/Archives/Public/public-whatwg-archive/2009Oct/0057.html) of `stepUp()` and `stepDown()` for `<input type="number">` based on the behavior of the spinner widget in __jQuery UI__. (from @scottgonzalez).
+
+# ECMAScript / TC39
+
+## Current proposals:
+
+| Proposal | Champion | Stage |
+|----------|----------|-------|
+| [String padding](https://github.com/tc39/proposal-string-pad-start-end) | Jordan Harband & Rick Waldron | 3 |
+| [Asynchronous Iterators](https://github.com/tc39/proposal-async-iteration) | [Kevin Smith](@zenparsing) | 2 |
+| [Observable](https://github.com/zenparsing/es-observable) | Kevin Smith & Jafar Husain | 1 |
+| [Private Fields](https://github.com/zenparsing/es-private-fields) | Kevin Smith | 1 |
+| [Class and Property Decorators](https://github.com/wycats/javascript-decorators/blob/master/README.md) | Yehuda Katz & Jonathan Turner | 1 |
+| [`String.prototype.at`](https://github.com/mathiasbynens/String.prototype.at) | Mathias Bynens & Rick Waldron | 0 |
+| [Function Bind Syntax](https://github.com/zenparsing/es-function-bind) | Kevin Smith | 0 |
+
+## Landed features:
+
+| Feature | Version | Champion |
+|---------|---------|----------|
+| Exponentiantion Operator | [ES2016](https://tc39.github.io/ecma262/2016/#sec-exp-operator) | [Rick Waldron](@rwaldron) |
+| [`Object.values` and `Object.entries`](https://github.com/tc39/proposal-object-values-entries) | [ES2016](https://tc39.github.io/ecma262/#sec-object.entries) | Jordan Harband (reviewed by @wycats @littledan @rwaldron) |
+
```

Comment:
I really enjoyed the discussion on it, at least: https://esdiscuss.org/topic/math-tau